runtime/lib/CMakeLists.txt

```diff
@@ -24,6 +24,14 @@ target_include_directories(TTBinary
 )
 add_dependencies(TTBinary FBS_GENERATION)
 
+add_library(TTRuntimeTypes STATIC types.cpp)
```

Comment:
Please add this static library (`TTRuntimeTypes`) to be the part of the shared lib used by tt-forge (in `lib/SharedLib/CMakeLists.txt`).